data/json/items/gun/45colt.json

```diff
@@ -96,5 +96,28 @@
     "proportional": { "reload": 1.5 },
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "45colt": 6 } } ],
     "melee_damage": { "bash": 8 }
+  },
+  {
+    "id": "colt_saa_old",
```

Comment:
This would be a variant in main game, it should be in the mod's files.

Reply:
I'm not quite sure I follow. I added to the main files as it seemed like it would be fine as a main game addition.

Reply:
Because in main game these two would have the exact same stats, they should have this on a single item's definition:
```
"variant_type": "gun",
    "variants": [
      {
        "id": "colt_saa_old",
        "name": { "str_sp": "Colt Single-Action Army Revolver" },
        "description": "This is an original antique of the legendary Colt Single Action Army, or Colt Peacemaker, one of the first revolvers to use a modern self-contained cartridge.  Made famous by Westerns, it is still in demand for Cowboy Action Shooting, reenactors, and collectors.  Unlike modern revolvers, the cylinder cannot swing out for loading, and spent brass must be ejected one at a time."
        "weight": 1
      },
      {
        "id": "uberti",
        "name": { "str": "Uberti Cattleman" },
        "description": "This 7.5\" barreled Uberti Cattleman is a modern reproduction of the legendary Colt Single Action Army, or Colt Peacemaker, one of the first revolvers to use a modern self-contained cartridge.  Made famous by Westerns, it is still in demand for Cowboy Action Shooting, reenactors, and collectors.  Unlike modern revolvers, the cylinder cannot swing out for loading, and spent brass must be ejected one at a time."
      }
    ],
    ```
    
    Flags can't change between variants, so having a version that has OLD_GUN on it requires two item definitions.

Reply:
Went ahead and removed it.